source/Patches/MeetingIntroPatch.cs

```diff
@@ -0,0 +1,39 @@
+﻿using HarmonyLib;
+using System.Collections.Generic;
+using UnityEngine;
+using PlayerData = GameData.PlayerInfo;
+using UnhollowerBaseLib;
+namespace TownOfUs
+{
+    
+    public static class MeetingIntroPatch
+    {
+        public static List<PlayerData> DeadPlayers =
+            new List<PlayerData>();
+
+        [HarmonyPatch(typeof(MeetingHud), nameof(MeetingHud.Start))]
+        public static class MeetingHud_Start
+        {
+            public static void Postfix() =>
+                Utils.ShowDeadBodies = PlayerControl.LocalPlayer.Data.IsDead;
+        }
+
+        [HarmonyPatch(typeof(MeetingIntroAnimation), nameof(MeetingIntroAnimation.CoRun))]
+        public static class IntroPatch
+        {
+            public static void Prefix(
+                [HarmonyArgument(1)] ref Il2CppReferenceArray<PlayerData> deadPlayers
+            )
+            {
+                var players = new List<PlayerData>(deadPlayers);
```

Comment:
Might as well be a HashSet
```cs
var players = deadPlayers.ToHashSet();
players.UnionWith(DeadPlayers);
deadPlayers = players.ToArray();
```

Reply:
i don't see any `ToHashSet` method nor a `ToArray` method on `HashSet<T>`? perhaps i am missing an import?

Reply:
`using System.Linq;`

Reply:
This should actually be done with `var players = new HashSet(deadPlayers, Il2CppEqualityComparer<PlayerData>.Instance);` because unhollowed objects don't override `Equals` (so collection conditions like `Contains` will incorrectly return `false`), `Il2CppEqualityComparer` is part of the Reactor API, probably needs to be readded since ToU doesn't use it. https://github.com/NuclearPowered/Reactor/blob/master/Reactor/Il2CppEqualityComparer.cs

Reply:
on second thoughts i don't think there should ever even be duplicate entries

Reply:
Might be if someone dies twice in the same round by being revived either by time keeper or altruist.

Reply:
they would have to be revived after their body was cleaned, so altruist couldn't revive them, i suppose its possible they could die twice due to time lord reviving them